templates/NOTES.txt

```diff
@@ -8,9 +8,13 @@ Congratulations! You have just deployed StackStorm {{ if .Values.enterprise.enab
   ╚══════╝   ╚═╝   ╚══════╝    ╚═╝  ╚═╝╚═╝  ╚═╝     ╚═════╝ ╚═╝  ╚═╝
 
 1. Get the StackStorm Web UI URL by running these commands:
+{{- if ne .Values.st2web.service.type "LoadBalancer" }}
```

Comment:
Need more cases for other Service types, as we now open this setting in values.yaml and expect that `NOTES.txt` will show service endpoint correctly.